sql/core/src/main/scala/org/apache/spark/sql/catalyst/util/V2ExpressionBuilder.scala

```diff
@@ -90,6 +93,8 @@ class V2ExpressionBuilder(e: Expression, isPredicate: Boolean = false) {
       }
     case Cast(child, dataType, _, true) =>
       generateExpression(child).map(v => new V2Cast(v, dataType))
+    case AggregateExpression(aggregateFunction, _, isDistinct, None, _) =>
```

Comment:
since we only translate expressions in logical plan to v2, the agg mode should always be `Complete`.
```suggestion
    case AggregateExpression(aggregateFunction, Complete, isDistinct, None, _) =>
```